### PR TITLE
Statistics and changelogs for zeus-v2021.07 release

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by lluiscampos** [comment]
I haven't pushed yet the tag, waiting for review.

File: 40.meta-mender/docs.md [hunk 1]
```diff
@@ -7,6 +7,98 @@ shortcode-core:
...
+
+| Top lines changed by employer | |
+|---|---|
+| Northern.tech | 1315 (99.8%) |
+| texier.pj2@gmail.com | 1 (0.1%) |
+| contact@coreycothrum.com | 1 (0.1%) |
+
+| Employers with the most signoffs (total 6) | |
+|---|---|
+| Northern.tech | 6 (100.0%) |
```
Comment (merlin-northern) [inline 40.meta-mender/docs.md:56]: why only 1 of 6 we show?

File: 40.meta-mender/docs.md [hunk 2]
```diff
@@ -7,6 +7,98 @@ shortcode-core:
...
+* mender-client-dev package to distribute D-Bus API specs
+  ([MEN-4106](https://tracker.mender.io/browse/MEN-4106))
+* fix grub chownboot task ordering
+* Add io.mender.UpdateManager to D-Bus policy files.
+* Fix build error when trying to build mender-configure
+  without systemd:
+  ```
+  mender-configure-1.0.0-r0 do_package: SYSTEMD_SERVICE_mender-configure value mender-configure-apply-device-config.service does not exist.
+  ```
+* u-boot-mender: Ignore case when checking variables in mender_Kconfig_fragment
```
Comment (merlin-northern) [inline 40.meta-mender/docs.md:80]: suggested change "Ignore case" -> "Ignore _the_ case" (or is it taken from the commit messages?)

**Review by merlin-northern — APPROVED** [review]
lgtm

File: 40.meta-mender/docs.md (hunk 1 above)
Comment (lluiscampos) [reply 40.meta-mender/docs.md:56]: That means that 6 signoffs (100%) are coming from NT

File: 40.meta-mender/docs.md (hunk 2 above)
Comment (lluiscampos) [reply 40.meta-mender/docs.md:80]: I cannot rephrase a commit message :) Only in very serious typos I would edit the auto-generated text

**Pull request merged**